### PR TITLE
Bandage overlays

### PR DESCRIPTION
**Pull request by alsoandanswer**

### CODE REVIEW

**Comment by KasparoVy** [comment]
Lmfao now I want to see a mummy. Someone just absolutely done up with bandages.

**Review by Chada-1 — APPROVED** [review]

**Comment by alsoandanswer** [comment]
!wip

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/stacks/medical.dm [hunk 1]
```diff
@@ -157,6 +157,7 @@ Contains:
 					else
 						to_chat(user, "<span class='warning'>\The [src] is used up, but there are more wounds to treat on \the [affecting.name].</span>")
 				use(used)
+				H.update_bandages(1)
```
Comment (Geevies) [inline code/game/objects/items/stacks/medical.dm:160]:
```suggestion
				H.update_bandages(TRUE)
```

File: code/game/objects/items/stacks/medical.dm [hunk 2]
```diff
@@ -282,6 +283,7 @@ Contains:
...
```
Comment (Geevies) [inline code/game/objects/items/stacks/medical.dm:286]:
```suggestion
				H.update_bandages(TRUE)
```

File: code/modules/mob/living/carbon/human/species/station/tajara/tajara.dm [hunk 3]
```diff
@@ -109,4 +110,4 @@
 		return
 	var/obj/item/clothing/shoes/sandal/S = new /obj/item/clothing/shoes/sandal(H)
 	if(H.equip_to_slot_or_del(S,slot_shoes))
-		S.autodrobe_no_remove = 1
\ No newline at end of file
+		S.autodrobe_no_remove = 1
```
Comment (Geevies) [inline code/modules/mob/living/carbon/human/species/station/tajara/tajara.dm:113]:
```suggestion
		S.autodrobe_no_remove = TRUE
```

File: code/modules/mob/living/carbon/human/species/station/unathi/unathi.dm [hunk 4]
```diff
@@ -114,4 +115,4 @@
 		return
 	var/obj/item/clothing/shoes/sandal/S = new /obj/item/clothing/shoes/sandal(H)
 	if(H.equip_to_slot_or_del(S,slot_shoes))
-		S.autodrobe_no_remove = 1
\ No newline at end of file
+		S.autodrobe_no_remove = 1
```
Comment (Geevies) [inline code/modules/mob/living/carbon/human/species/station/unathi/unathi.dm:118]:
```suggestion
		S.autodrobe_no_remove = TRUE
```

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 5]
```diff
@@ -221,6 +221,24 @@ There are several things that need to be remembered:
 		LAZYADD(ovr, DI)
 
 	overlays_raw[DAMAGE_LAYER] = ovr
+	update_bandages(update_icons)
+	if(update_icons)
+		update_icon()
+
+/mob/living/carbon/human/proc/update_bandages(var/update_icons=1)
```
Comment (Geevies) [inline code/modules/mob/living/carbon/human/update_icons.dm:228]:
```suggestion
/mob/living/carbon/human/proc/update_bandages(var/update_icons = TRUE)
```

**Review by Geevies — APPROVED** [review]

**Pull request merged**